src/apps/search-result/search-result.tsx

```diff
@@ -184,7 +202,7 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
 
       {!isLoading && !shouldShowZeroHits() && resultItems && (
         <>
-          <SearchResultHeader hitcount={hitcount} q={q} />
+          <SearchResultHeader hitcount={hitcount} q={displayQuery} />
```

Comment:
We have to make sure that we have a clear concept between displaying a query and executing a query.
It seems like `SearchResultHeader`is recieving  `displayQuery ` but the prop on the component is still called `q`. While the `FacetLine ` component is also having a `q` prop which recieves `q`. 
That is a code smell to me :)

Reply:
Note: Change "q" to "displayQuery"